cypress/e2e/brewery/TableParameters-files_part2.cy.js

```diff
@@ -175,4 +175,32 @@ describe('Table parameters files standard operations part 2', () => {
     BreweryParameters.getCustomersTableCell('name', 4).should('have.text', 'Dwight');
     BreweryParameters.getCustomersTableCell('name', 5).should('have.text', 'Arnold');
   });
+
+  it('can add new lines in an empty table, save the changes and retrieve them after a page refresh', () => {
+    const scenarioName = forgeScenarioName();
+    scenarioNamesToDelete.push(scenarioName);
+    Scenarios.createScenario(scenarioName, true, SCENARIO_DATASET, SCENARIO_RUN_TEMPLATE).then((value) => {
+      const scenarioId = value.scenarioCreatedId;
+      const scenarioUrl = `${BREWERY_WORKSPACE_ID}/scenario/${scenarioId}`;
+
+      ScenarioParameters.expandParametersAccordion();
+      BreweryParameters.switchToCustomersTab();
+      BreweryParameters.getCustomersTableGrid().should('exist');
+
+      BreweryParameters.addRowCustomersTableData();
+      BreweryParameters.getCustomersTableRows().should('have.length', 1);
+      BreweryParameters.getCustomersTableCell('name', 0).should('have.text', 'value');
+      BreweryParameters.getCustomersTableCell('canDrinkAlcohol', 0).should('have.text', 'false');
+      BreweryParameters.getCustomersTableCell('age', 0).should('have.text', '');
+
+      ScenarioParameters.save();
+      route.browse({ url: scenarioUrl });
```

Comment:
Why don't use _cy.reload()_ ?

Reply:
`cy.reload` won't trigger all the interceptions that have been set in our `route` utils. These interceptions are required when using the stubbing, or to intercept queries we don't to send during tests (e.g. for the PowerBI token)